R/get-based-on.R

```diff
@@ -49,9 +62,7 @@ get_based_on.default <- function(.bbi_object, .check_exists = FALSE) {
   return(as.character(fs::path_norm(file.path(.bbi_object[[WORKING_DIR]], .bbi_object[[YAML_BASED_ON]]))))
 }
 
-
-#' @rdname get_based_on
-#' @param .bbi_object Character scaler of a path to a model that can be loaded with `read_model(.bbi_object)`
+#' @describeIn get_based_on Takes a character scaler of a path to a model that can be loaded with `read_model(.bbi_object)`.
```

Comment:
Typo: "scaler."

Reply:
You would think I would've just grepped for this by now...